ion_hash_tests.ion

```diff
@@ -0,0 +1,1631 @@
+/*
```

Comment:
I assume this is a temporary file? It would be better to add the tests from https://github.com/amzn/ion-hash-test as a git submodule.

Reply:
Done.